### DISCUSSION TITLE
Ideen für die weitere Entwicklung

### DISCUSSION THREAD

**Discussion by JKoblitz** [OSIRIS-Solutions/osiris#13] [Ideas]
Ihr wollt neue Features haben, habt tolle Ideen oder fragt euch, ob man irgendwas nicht auch besser machen kann? Immer her damit. Hier könnt ihr jede noch so wahnwitzige Idee mit der Community teilen. Und wenn eine Idee gut ist, machen wir daraus ein Issue und nehmen es in die Roadmap mit auf 😄.

**Comment by lvisch** [comment]
Wir überlegen gerade, wie wir unsere Patente in Osiris bekommen und haben gesehen, dass das europäische Patentamt eine API anbietet. Es gibt eine kostenlose Version ("Up to 4 GB of data per week"). Infos zu Patenten müssen ja auch an Leibniz gemeldet werden, es gibt also wahrscheinlich an mehreren Instituten Bedarf in die Richtung.
https://worldwide.espacenet.com/patent/
https://developers.epo.org/
Wäre das was für euch?
